src/meshapi/views/forms.py

```diff
@@ -134,6 +138,50 @@ def join_form(request: Request) -> Response:
             {"detail": "Your address could not be validated."}, status=status.HTTP_500_INTERNAL_SERVER_ERROR
         )
 
+    changed_info: dict[str, str | int] = {}
+
+    # TODO: Notify member if we changed any of their information
+    # Name (won't touch), email (won't touch), phone, st addr, unit (won't touch), city, State, Zip
+    if formatted_phone_number and r.phone != formatted_phone_number:
```

Comment:
Hmm so if they type `(555) 555-5555` and we reformat it to `+1 555-555-5555`, they will get a pop-up warning them we've made a change? Perhaps we should use a frontend validation library to format the phone number into a consistent format to prevent this

Reply:
We're still gonna want to do some validation in the backend, I think. I'm trying to get the behavior to be consistent. See my comments from this morning.

I mean, maybe we just silently accept whatever formatting our backend does. The main concern here is the address.

Reply:
We now have frontend validation for the phone number. It's still validated in the backend, but this should not be a concern.